tutorials/migrations/0002_auto_20191212_1122.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.24 on 2019-12-12 10:22
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('tutorials', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='tutorial',
+            name='category',
+            field=models.CharField(blank=True, choices=[('io', 'I/O'), ('intro', 'Introduction')], max_length=15),
+        ),
+        migrations.AddField(
+            model_name='tutorial',
+            name='level',
+            field=models.IntegerField(choices=[(1, '1'), (2, '2'), (3, '3')], null=True),
```

Comment:
What are levels 1-3 mean? I think we should add some natural language here instead of numbers.

Reply:
The levels come from the mockup: https://user-images.githubusercontent.com/7835111/68846472-6faea400-06cd-11ea-953f-1910490b47fb.png
I think there will be a icon based on each level value?

Reply:
Indeed, but even in the mockup there are natural language equivalents (Beginners, Intermediates, Advanced). I think we should use those formulations